frontend/packages/metal3-plugin/src/k8s/objects/bare-metal-host.ts

```diff
@@ -13,8 +12,8 @@ export const buildBareMetalHostSecret = (name, namespace, username, password) =>
     name: getSecretName(name),
   },
   data: {
-    username: Base64.encode(username),
-    password: Base64.encode(password),
+    username: btoa(username),
```

Comment:
I think I used `Base64.encode` to be in sync with the way it is done in the secrets codebase, but no big deal.

Reply:
I used these because it is more straightforward solution